.editorconfig

```diff
@@ -0,0 +1,64 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Common editor configurations for this project.
+#
+# EditorConfig is a file format for specifying some common style parameters.
+# Many IDEs & editors read .editorconfig files, either natively or via plugins.
+# We mostly follow Google's style guides (https://google.github.io/styleguide/)
+# with only a few deviations for line length and indentation in some files.
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
+insert_final_newline = true
+spelling_language = en-US
+trim_trailing_whitespace = true
+max_line_length = 120
+
+[{BUILD,*.BUILD,*.bzl,*.bazel,.bazelrc}]
+# Google doesn't have a style guideline for Bazel files. Most people use 4.
+indent_size = 4
+
+[{*.cc,*.h}]
+# Google style guidelines use 2.
+indent_size = 4
+
+[{*.js,*.ts}]
+# Google style guidelines use 2.
+indent_size = 4
+
+[*.json]
+# Not stated explicitly in Google's guidelines, but the examples use 2.
+indent_size = 2
+
+[*.py]
+# Google style guidelines use 4.
+indent_size = 4
+
+[*.rst]
```

Comment:
There aren't any rst files in the project?

Reply:
> There aren't any rst files in the project?

I did a `find . -name '*.rst'` and it turned up empty. Are there some?

Reply:
Not that I'm aware of